src/System.Windows.Forms/src/System/Windows/Forms/Application.ThreadContext.cs

```
@@ -1256,10 +1265,13 @@ internal bool ProcessFilters(ref User32.MSG msg, out bool modified)
                 // should not update the snapshot until the next message.
                 if (_messageFilters is not null && !GetState(STATE_FILTERSNAPSHOTVALID) && _inProcessFilters == 0)
                 {
-                    _messageFilterSnapshot.Clear();
-                    if (_messageFilters.Count > 0)
+                    if (_messageFilterSnapshot is not null)
```

This null-check is hiding an error that used to be reported.

---

`_messageFilters` and `_messageFilterSnapshot` are initialised simultaneously:
https://github.com/dotnet/winforms/blob/75dee0b5ba4ce9f1f433598c2aca7b1701f16893/src/System.Windows.Forms/src/System/Windows/Forms/Application.ThreadContext.cs#L386-L396

There are no other places where either of these is initialised or nulled out.